### DISCUSSION TITLE
Disable Phase One

### DISCUSSION THREAD

**Discussion by eeriemyxi** [ggandor/leap.nvim#209] [Q&A] [answered]
When I would type `sx` where x is some character, it highlights all the matches immediately, labeling each with some key _but_ if I press that key, it won't jump me there but go instead just do `sxy` where y is the second character I typed. This is extremely confusing and I don't even want to know the correct way to do it. I don't want to think about all this when I am doing it. Can I just make it wait with no visual hints whatsoever until I am done pressing two characters?

https://github.com/ggandor/leap.nvim/assets/86738916/b37a189b-2d85-414c-a837-a4103522f0fa

**Comment by ggandor** [comment, marked as answer]
`require('leap').opts.max_phase_one_targets = 0`

https://github.com/ggandor/leap.nvim/blob/bf5d64f072a74968191f9994b0c05fac64f5f015/doc/leap.txt#L202-L207
